parts/linux/cloud-init/artifacts/components.json

```diff
@@ -359,14 +359,17 @@
         "v1.28.9-1",
         "v1.28.11",
         "v1.28.12",
+        "v1.28.13",
```

Comment:
can we start thinking about removing removing 1.27 patch versions? we need to be cognizant of the space requirements for caching all these versions at once

Reply:
no, we have to keep 1.27 related container images for LTS customers.

Reply:
but I don't think we need this much:
  ~~"v1.27.13",~~
  ~~"v1.27.13-2",~~
  ~~"v1.27.14-1",~~
  "v1.27.15",
   "v1.27.16",

Reply:
I'll merge this pr first. we might discuss it later and make necessary changes in rp